packages/databricks-vscode/src/bundle/models/BundlePreValidateModel.ts

```diff
@@ -67,16 +78,30 @@ export class BundlePreValidateModel extends BaseModelWithStateCache<BundlePreVal
             : undefined;
     }
 
+    private async getRawTargetData(target: string) {
+        const bundle = await this.bundleFileSet.bundleDataCache.value;
```

Comment:
Can we pass bundle data as an argument and make this function synchronous? I think it will make the other code a bit more readable